src/renderer/notification.ts

```diff
@@ -655,6 +655,7 @@ class Notification extends NotificationHandler {
       frame: false,
       transparent: true,
       fullscreenable: false,
+      type: 'toolbar',
```

Comment:
`toolbar` is only supported on Windows. Does this work on MacOS too?

Reply:
we have already handled it below, as 'Panel'